.expeditor/config.yml

```diff
@@ -0,0 +1,44 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: sustaining-notify
+
+rubygems:
+  - knife-azure
+
+github:
+  # This deletes the GitHub PR branch after successfully merged into the release branch
+  delete_branch_on_merge: true
+  # The tag format to use (e.g. v1.0.0)
+  version_tag_format: "v{{version}}"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/msys-developers
+  # Which Github branches to build gems from, and what versions
+  # (as determined by the value in the VERSION file) those branches are responsible
+  # for building.
+  release_branch:
+    - master:
+        version_constraint: 1.*
```

Comment:
since we're not doing multiple branches, I think we can delete this whole release_branch section (lines 20-22)

Reply:
Thanks @btm, I have removed `release_branch` section from config.yml